packages/types/src/dtos/posts/post.dto.ts

```diff
@@ -63,7 +63,7 @@ export class Post {
   @IsNotEmpty()
   @IsNumber({ maxDecimalPlaces: 2 })
   @Max(99999999.99)
```

Comment:
Maximum per charge of omise 150,000 THB

Reply:
In that case, let us divide to multi transaction.